docs/source/using_models.rst

```diff
@@ -295,4 +295,35 @@ For reference, here is the JSON export for the same `Document` as above::
             ]
         }
     ]
-  }
\ No newline at end of file
+  }
+
+To export the outpout as XML (hocr-format) you can use the `export_as_xml` method::
+
+  xml_output = result.export_as_xml()
+  for output in xml_output:
+    xml_bytes_string = output[0]
+    xml_element = output[1]
+
+For reference, here is a short snippet of the XML output::
```

Comment:
"For reference, here is a sample XML output" might be better

And actually, I don't even know: is this the bytes string? or the element?

Reply:
@fg-mindee 
that is the bytes string as prettyprint

Reply:
Let's reflect that then "For reference, here is a sample XML byte string output"